test/agent_writer_test.cpp

```diff
@@ -282,8 +282,8 @@ TEST_CASE("writer") {
   SECTION("failed agent comms") {
     std::unique_ptr<MockHandle> handle_ptr{new MockHandle{}};
     MockHandle* handle = handle_ptr.get();
-    std::vector<std::chrono::milliseconds> retry_periods{std::chrono::milliseconds(500),
-                                                         std::chrono::milliseconds(2500)};
```

Comment:
ahaha, got bored of waiting for the tests huh? :D

Reply:
Just a bit.